src/System.Diagnostics.TraceSource/tests/TestTraceListener.cs

```diff
@@ -119,5 +119,10 @@ public override void WriteLine(string message)
         {
             Call(Method.WriteLine);
         }
+
+        public override void Close()
+        {
+            Dispose(); // In desktop Trace.Close() calls listener.Close() on every listener and in Core we call listener.Dispose() so the tests would fail in desktop without this override. 
```

Comment:
We could do that, would you prefer me to open an issue to fix that in desktop and skip the test in desktop?